app/waf/src/models/application.model.ts

```diff
@@ -12,13 +12,82 @@ export class Application extends CommonEntity {
   })
   status?: string;
 
+  @property({
+    type: 'string',
+    required: false,
+    schema: {
+      create: true,
+      update: true,
+      response: true,
+      example: '11111111-2222-3333-4444-555555555555',
+    },
+  })
+  adcId?: string;
```

Comment:
Adc and application can be decoupled. I think there's no need to make an adcId in application. 

Suppose users deploy their configuration by running `deploy(adcId, declarationId)`, it's simple.

Reply:
The deploy API need to support an adcId parameter, in order to deploy the configuration to a user specified device. It also need a default device id, so that user can deploy the config without inputing a device id.